qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -69,6 +69,10 @@
                             cpu_model = host
                         - default_model:
                             cpu_model_flags = ",+serialize,+avx512-vp2intersect,+avx512-fp16,+tsxldtr"
+                - avx_vnni:
+                    required_qemu = [6.0.0-26, )
```

Comment:
I guess we need add guest restriction as well. Old kernel may not support it?

Reply:
@nanliu-r Any update for this ?

Reply:
> @nanliu-r Any update for this ?

Seems old guests support in cpuid, maybe we should add a function to check cpuid.

Reply:
> I guess we need add guest restriction as well. Old kernel may not support it?
confirmed with Yumei, and the guest kernel won't see the flag if it doesn't support, update the the guest types, thanks.